### PR TITLE
added Slowhttptest tool

### PR DESCRIPTION
**Pull request by Bipan079**

### CODE REVIEW

File: src-tauri/tauri.conf.json [hunk 1]
```diff
@@ -257,6 +257,11 @@
                         "cmd": "sherlock",
                         "args": true
                     },
+                    {
+                        "name": "slowhttptest",
+                        "cmd": "slowhttptest",
+                        "args": true
+                    }
```
Comment (basoreeper3007) [inline src-tauri/tauri.conf.json:264]:
```suggestion
                    },

File: src/components/RouteWrapper.tsx [hunk 2]
```diff
@@ -607,6 +608,13 @@ export const ROUTES: RouteProperties[] = [
             "Shodan API allows for network scanning and querying of Shodan’s database to gather information about connected devices and their vulnerabilities.",
         category: "Vulnerability Assessment and Exploitation",
     },
+    {
+        name: "SlowHttpTest",
+        path: "/tools/slowhttptest",
+        element: <SlowHttpTest />, // Add SlowHttpTest component
```
Comment (basoreeper3007) [inline src/components/RouteWrapper.tsx:614]: This inline comment adds no value to the code.

```suggestion
        element: <SlowHttpTest />,

**Review by basoreeper3007 — COMMENTED** [review]
Code looks pretty good. Please make the suggested changes. Also make sure commit messages reflect what you have changed in code rather than using generic "Updated Code" commit messages.

**Review by Siddh2001 — APPROVED** [review]
Good job @Bipan079 
Your work looks good to me

**Pull request merged**